public/router.js

```diff
@@ -0,0 +1,17 @@
+//Criação da rota
+
+//Exemplo var path __dirname : http://stackoverflow.com/questions/7083045/fs-how-do-i-locate-a-parent-folder
+
+var path = require('path');
+path.resolve = __dirname + "/public/";
+
+module.exports = function(app) {
+   app.get("/", function(request, response) {
+       response.sendFile(path + "index.html");
+   });
+
+  app.get("/api/students/:id", function (request, response) {
+        const id = request.params.id;
+    });
+};
+
```

Comment:
You and @gustavomartins95 have used the same approch. Where did you guys get it? Even the space are the same!!

**Your score will be blocked until we clarify it!!**

![image](https://cloud.githubusercontent.com/assets/5324414/21125591/03441374-c0ce-11e6-8d65-a6cda18bdab0.png)